lxdock/provisioners/ansible.py

```diff
@@ -40,10 +40,31 @@ def provision(self):
             self.host.run(self._build_ansible_playbook_command_args(tmpinv.name))
 
     def setup_guest_alpine(self):
-        # On alpine guests we have to ensure that ssd is started!
+        # On alpine guests we have to ensure that sshd is started!
         self.guest.run(['rc-update', 'add', 'sshd'])
         self.guest.run(['/etc/init.d/sshd', 'start'])
 
+    def setup_guest_arch(self):
+        # On archlinux guests we have to ensure that sshd is started!
+        # https://bbs.archlinux.org/viewtopic.php?id=205367
+        self.guest.run(['systemctl', 'enable', 'sshd.socket', '--now'])
```

Comment:
Question: I am repeating this "ensure sshd" code in both `ansible.py` and `puppet.py`. Is there a better way to do this? For example, I was also thinking to put it as `ArchLinuxGuest.ensure_sshd()`.